src/game.cpp

```diff
@@ -609,6 +609,14 @@ bool Game::removeCreature(Creature* creature, bool isLogout/* = true*/)
 	return true;
 }
 
+void Game::executeDeath(uint32_t creatureId)
+{
+	Creature* creature = getCreatureByID(creatureId);
+	if (creature && !creature->isRemoved() && creature->getHealth() < 1) {
```

Comment:
checking life again? I think it is not necessary.

Reply:
changed